packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -61,7 +64,7 @@ class BottomNavigationBarItem {
   final Widget activeIcon;
 
   /// The title of the item. If the title is not provided only the icon will be shown when not used in a Material Design [BottomNavigationBar].
-  final Widget title;
+  final Text title;
```

Comment:
Since the recommendation doesn't depend on this being a `Text` widget, undo this change and its consequences.

Reply:
@willlarche This is appropriate to get the text data out of `Text` widget. If some other widgets can wrap the `Text` widget like below
```
Container(child: Text(....
```
It is hard to get the text data out of it which should be used in `semanticLabel`.

Reply:
This is a large breaking change. It wouldn't be possible to provide a RichText or another widget which wrapped text.

With the use of opacity above it isn't necessary to pull the semanticLabel out manually

Reply:
@jonahwilliams Thanks so much for the clarification. You've made it clear and I've reverted this.